clients/flash/common-library/src/main/actionscript/org/bigbluebutton/lib/common/views/ParticipantIcon.mxml

```diff
@@ -5,36 +5,29 @@
 
 	<fx:Script>
 		<![CDATA[
-			public function setFirstLetters(name:String):void {
-				if (name == null) name = "";
-				
-				var names:Array = name.split(" ");
-				var firstLettersText:String = names[0].charAt(0);
-				if (names[1]) {
-					firstLettersText += names[1].charAt(0);
-				}
-				firstLetters.text = firstLettersText.toUpperCase();
-			}
+			[Bindable]
+			public var displayInitials:String;
 		]]>
 	</fx:Script>
 
 	<s:Label id="firstLetters"
-			 fontSize="{getStyle('fontSize')}"
```

Comment:
The font size has been moved to stylesheet files. It can be found under style class `participantIcon`.